### DISCUSSION TITLE
US06 drive cycle related

### DISCUSSION THREAD

**Discussion by alihassan90** [pybamm-team/PyBaMM#3137] [General]
The following CSV file is based on US06 drive cycle, contains the currents (+ive and -ive) which I believe are the per second charge/discharge of battery pack. However, I failed to find out which cell/battery pack are these currents for? i.e energy rating, battery make etc. 

https://github.com/pybamm-team/PyBaMM/blob/develop/pybamm/input/drive_cycles/US06.csv?plain=1

**Comment by brosaplanella** [comment]
This is an arbitrary drive cycle, so not defined for a specific cell. However, you will most likely have to rescale the current so it is suitable to the cell you are simulated. To do so, you can just rescale the second column (one way is to rescale so the maximum current has a certain value, e.g. 2C).

**Reply by alihassan90** [reply]
Thank you for help. Is there a reference for this drive cycle? Like the original source, because I have to refer it in paper. I found a website https://www.epa.gov/vehicle-and-fuel-emissions-testing/dynamometer-drive-schedules but there US06 is in terms of mph and not the current [A]. How was current determined from mph.

**Reply by brosaplanella** [reply]
I don't know what the source for the drive cycle is, it is just provided as an example.
